File: spec/code_samples/Python_python3/auth/post.py
import http.client

conn = http.client.HTTPConnection("example.com")

payload = "{\"email\":\"<ADD STRING VALUE>\",\"password\":\"<ADD STRING VALUE>\"}"

headers = {
    'content-type': "application/json",
    'authorization': "Bearer REPLACE_BEARER_TOKEN"
    }

conn.request("POST", "/api/v1/auth", payload, headers)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))

File: spec/code_samples/Node_request/auth/post.js
var request = require("request");

var options = { method: 'POST',
  url: 'https://cleanse.studio/api/v1/auth',
  headers: { 'content-type': 'application/json' },
  body:
   { email: '<ADD STRING VALUE>', password: '<ADD STRING VALUE>' },
  json: true };

request(options, function (error, response, body) {
  if (error) throw new Error(error);

  console.log(body);
});
